electron/src/preload/preload-app.ts

```diff
@@ -18,19 +18,200 @@
  */
 
 import {contextBridge, ipcRenderer, webFrame} from 'electron';
-import {truncate} from 'lodash';
 
-import * as path from 'path';
+// Context Isolation Security: Constants and utilities must be defined inline
+// because sandboxed preload scripts cannot import from relative paths due to
+// limited CommonJS module resolution in Electron's sandbox environment.
+// These are kept in sync with ../shared/contextIsolationConstants.ts
 
-import {WebAppEvents} from '@wireapp/webapp-events';
+/**
+ * Event type constants for IPC communication
+ *
+ * IMPORTANT: This is a necessary duplication of ../shared/contextIsolationConstants.ts
+ * due to Electron sandbox limitations. Keep in sync with the shared file.
+ */
+const EVENT_TYPE = {
```

Comment:
no one will remember to keep this in sync with the package. 
it also defeats the use case of the package. 

This needs to be done automated, if the contents need to be copied over from the package, then there should be a sync command that does it automatically aka "yarn sync:events"